readthedocs/proxito/tests/test_full.py

```diff
@@ -35,6 +35,13 @@
 class TestFullDocServing(BaseDocServing):
     # Test the full range of possible doc URL's
 
+    def test_health_check(self):
+        url = '/_health_check/'
+        host = 'project.dev.readthedocs.io'
```

Comment:
The health check from the ALB comes in without a host header. Doesn't seem like it would be too much difference, but in case the host name could determine the view being used, maybe best to test without